example/src/theme/ApplicationStyle.ts

```diff
@@ -1,5 +1,4 @@
 import { StyleSheet } from 'react-native';
-import { Metrics } from 'react-native-animation-catalog';
```

Comment:
@het-a-simform Hope removed import is unused in the file

Reply:
Yes removed import is unused.